remote-api/src/main/java/org/alfresco/rest/api/Aspects.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2016 Alfresco Software Limited
```

Comment:
Please update the copyright year from 2016 to 2021, same comment for all new files below.